Automattic/wp-calypso
# add VideoPress product to jetpack Complete "included" products

#### Changes proposed in this Pull Request

* Add "Jetpack VideoPress" product to the list of included features for "Jetpack Complete" so it gets flagged correctly as being included in the plan

**For a Jetpack site with Jetpack Complete Monthly plan**

**Before**
<img width="1089" alt="Screen Shot 2021-12-20 at 1 50 20 PM" src="https://user-images.githubusercontent.com/4081020/146817843-4e9b4ef6-13a8-47cd-865a-664a20ba254f.png">


**After**
<img width="1123" alt="Screen Shot 2021-12-20 at 1 48 33 PM" src="https://user-images.githubusercontent.com/4081020/146817860-f11f2be4-0d23-48a0-9dfa-137b7302785a.png">


#### Testing instructions


* Ensure `Jetpack VideoPress` is not purchased on your Jetpack site
* Add the `Jetpack Complete` **Monthly** plan
* Visit the Upgrades -> Plans menu and scroll down, VideoPress should be identified as "Part of your current plan"
* Jetpack Complete **Annual** does not appear to display the same screen, so it isn't exhibiting the same issue



Related to # 1057-gh-Automattic/greenhouse
